### ISSUE TITLE
False positive for PartialInstanceVariable in ERB comment?

### ISSUE THREAD

**Issue by steveclarke** [Shopify/erb-lint#315]
When my `.erb-lint.yml` contains:

```
linters:
  PartialInstanceVariable:
    enabled: true
```

and I have the following commented-out line in my `_form.html.erb` file:

`<%#= debug_json @employer.contracts %>`

`erblint app/views/configuration/employers/_form.html.erb`  reports:

```
Linting 1 files with 16 linters...

Instance variable detected in partial.
In file: app/views/configuration/employers/_form.html.erb:195
```

It's nice that erblint warns me that, should I uncomment the code, it would trigger an issue, but is this expected behaviour? I would have expected erblint to ignore anything in comments. What if I wanted to use that comment to say what *not* to write?

**Comment by sfnelson** [comment]
This false positive also shows up with @type annotations:

```
<%# locals: (item:) %>
<%# @type [RecordType] item %>
```

**Comment by sfnelson** [comment]
Also shows up in styles:

```
<style>
  @font-face {
     ...
   }
</style>
```
